AutoProvisionAndLogin.php

```diff
@@ -33,17 +33,26 @@ function sso_logout()
 
 function sso_login()
 {
-    //$email_address = explode("/", $_SERVER['REMOTE_USER'])[0];
-    //$user_number = explode("@", $email_address)[0];
-
-    if (is_user_logged_in()) return;
+    if (is_user_logged_in()) {
+        return;
+    }
 
+    /*
+    $email_address = explode("/", $_SERVER['REMOTE_USER'])[0];
+    $user_number = explode("@", $email_address)[0];
+    /*/
     $email_address = 'testmonkey@example.com';
     $user_number = 9999;
+    //*/
+
     if ($email_address == '') {
         echo("Did not obtain user info from SSO server");
         return;
-    } elseif (null == username_exists($user_number)) {
+    }
+
+    $user = get_user_by('login', $user_number);
+
+    if ($user == false) {
```

Comment:
I assume ($user == false) is equivalent/better than (null == username_exists($user_number)  ?